embeds.js

```diff
@@ -264,16 +270,18 @@ const embedPartnership = {
 
         return PartnershipEmbed;
     },
-    PartnershipRequester: async function PartnershipRequester(channel, enable, memberRequirement, roleMention){
+    PartnershipRequester: async function PartnershipRequester(channel, enable, memberRequirement, roleMention, questions){
         // Enable is true if the partnership module is enabled, False if edited
+        // questions is a array of questions from Custom Questions
+        
         let embed = new EmbedBuilder(embedInfo.Success)
             .setTitle(`${iconSuccess} PARTNERSHIPS ENABLED`)
             .setDescription(`The partnerships module has successfully been enabled, and was sent to <#${channel}>.\n\u200B`)
             .setThumbnail(iconURLCommunity)
             .setFooter(footerPartnership)
         return embed
     },
-    PartershipAccepted: async function PartershipAccepted(user){
+    PartnershipAccepted: async function partnershipAccepted(user){
```

Comment:
async function partnershipAccepted(user) is not capitalized, I will edit this after merge.